windows.md

```diff
@@ -390,6 +390,49 @@ With those compatibility things out of the way, you're ready to start the system
     4. Open the start menu and search for "Ubuntu". Start it - it should ask you to create a user with a password. This will be your user to log in to your Ubuntu Linux Subsystem - note down the username and password somewhere secure to make sure you do not forget it.
     5. Open the start menu and search for "Docker Desktop". Start it and go to the Settings. Under the General tab, you will find an option called "Use WSL 2 based engine". Make sure this is checked.
 
+    If opening Docker Desktop from **Option A or B** shows an error, you may need to enable virtualization on your machine
+
+    The error might look like this:<br><br>
+    <img src="docker-desktop-opening-error.avif"><br><br>
+
+    Close PowerShell and open it again as administrator (like in step 2)
+
+    Check if your computer supports virtualization. Run this command:
+
+    ```bash
+    Get-ComputerInfo -Property HyperV*
+    ```
+
+    This should display the following output:
+
+    ```bash
+    HyperVisorPresent: False
+    HyperVRequirementDataExecutionPreventionAvailable : True
+    HyperVRequirementSecondLevelAddressTranslation    : True
+    HyperVRequirementVirtualizationFirmwareEnabled    : False
+    HyperVRequirementVMMonitorModeExtensions          : True
+    ```
+
+    Check if virtualization is enabled. Run this command:
+
+    ```bash
+    systeminfo | findstr Virtualization
+    ```
+
+    This should display the following output:
+
+    ```bash
+    Virtualization Enabled In Firmware: No
+    ```
+
+    Continue with the steps below if virtualization is not enabled to enable virtualization in your BIOS/UEFI:
```

Comment:
- I don't think that it needs to say "continue" here, it's natural to keep following the steps if nothing goes wrong
- there should be some text to describe what to do if either of the "outputs" above are different (and what that means)
- maybe we still want to separate these steps to enable virtualization in BIOS / UEFI

Reply:
> maybe we still want to separate these steps to enable virtualization in BIOS / UEFI (but maybe it's not important to have them separated from step one "Close PowerShell ..." above)

I don't think we should make it a separate point as it is not something that all students would need to do. I think I did had this issue because I moved from Windows 10 to 11. According to [this docs](https://support.microsoft.com/en-us/windows/enable-virtualization-on-windows-11-pcs-c5578302-6e43-4b4b-a449-8ced115f58e1), it seems most computers should have it already enabled.

The Continue with the... statement is combined with the new one on the top as I think they should continue with anything on these steps only if they have an error

https://github.com/upleveled/system-setup/pull/16/commits/493e54b7da21449f98e031dc87aead474b1a121d
https://github.com/upleveled/system-setup/pull/16/commits/63fe30ab3bddc4701ef0e54ac5d1981c9ee84d24